app/lib/submission_builder/formatting_methods.rb

```diff
@@ -26,6 +26,17 @@ def date_type(date)
       date.strftime("%F")
     end
 
+    def date_type_for_timezone(date)
+      return nil unless date.present?
+      date = case @submission.data_source.state_code
+             when 'ny', 'md', 'nc', 'nj'
+               date.in_time_zone('America/New_York')
+             else
+               date.in_time_zone('America/Phoenix')
+             end
+      date.strftime("%F")
```

Comment:
to make this helper method more reusable, this formatting call should be removed and instead applied when it is called downstream